src/ElectricPokemon.java

```diff
@@ -0,0 +1,114 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private final String type = "electric";
+    private final List<String> attacks = Arrays.asList("Inferno", "PyroBall", "FireLash", "FlameThrower");
+
+    public ElectricPokemon() {
```

Comment:
Deze constructor is niet verstandig hier. JE wilt namelijk geen Pokemon zonder naam, level, hp, food en sound hebben. Dat is wel wat deze constructor doet. Je gebruikt deze constructor ook nergens